_includes/homepage-video-band.html

```diff
@@ -0,0 +1,11 @@
+<div class="full-width-bg component homepage-content-band video-band">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m video-container">
+      <iframe src="https://www.youtube.com/embed/hhHgurtI674" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
Its nice with a introductory video but the tone/content of it don't really send out a vide of "this project is interesting and new". 

I would postpone adding such content until we have a video with a bit more developer centric and to the point story.

Reply:
I agree, I think this video is better suited for the upcoming RHT Quarkus website which will be more business minded.